### PR TITLE
Add docs, pypi and license badge

### PR DESCRIPTION
**Pull request by jostosh**
- Add docs, pypi and license badge to README.md

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# .github/workflows/test-pypi.yml
```
Comment (lstout) [inline .github/workflows/tests.yml:1]: Very minor thingy

```suggestion
# .github/workflows/tests.yml
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -60,12 +37,7 @@ jobs:
       run: poetry install
 
     - name: Run pytest
-      run: poetry run coverage run --source ./src/grpc_argument_validator -m unittest discover -s src/tests -p '*_test.py'
-
-    - name: Upload Coverage Report
-      run: |
-        poetry run coverage xml --fail-under=0
-        poetry run codecov
+      run: poetry run python -m unittest discover -s src/tests -p '*_test.py'
```
Comment (lstout) [inline .github/workflows/release.yml:40]: Is this job still necessary in the release workflow?

**Review by lstout — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 2 above)
Comment (jostosh) [reply .github/workflows/release.yml:40]: The test badge will be named using `<workflow>.name`, which is why I separated `Tests` from `Release` in the first place. 

It is AFAIK impossible to let one workflow depend on another, so these tests are there to guarantee that we cannot release a package for which tests fail. So I agree, that it seems redundant to have tests here, but I cannot find a better alternative as of now.

Comment (lstout) [reply .github/workflows/release.yml:40]: Cool :)

**Pull request merged**